### PR TITLE
fixed user.js in case of no profile pic

### LINKED ISSUE #261: fetching an undefined resource
The console shows a 404 error for logged-in users without a profile picture.

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
### Issue
If the user has no Profile picture the console throws a 404 error

### What is the change?

I changed the user.js to check if the profile image is present or not if yes render it else render the default profile logo

##### Explaination 
When I sign in using Github the profile pic is not provided so the navbar loads no image and searches for an undefined resource throwing an error in the console. 

- Done this: added 2 lines in user.js
- Made these changes: used if else case in lines 22 and 23

### Is Development Tested?

-  Yes

#### Before
image

#### After 
image

### CODE REVIEW

File: js/user.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ const setUserGreeting = (username, firstName, userProfilePicture) => {
     const userImgURL = userProfilePicture;
 
     userImgEl.forEach((element) => {
-      element.src = userImgURL;
+      if (!!userImgURL) element.src = userImgURL;
```
Comment (MehulKChaudhari) [inline js/user.js:22]: Why do `!!userImgURL`? simply `userImgURL` will work in the if-case, its readable and understandable

Comment (MehulKChaudhari) [reply js/user.js:22]: also, use proper `if(){} else{}` syntax its also easy to understand.

**Review by MehulKChaudhari — COMMENTED** [review]
Add a few neat comments. Please do check

File: js/user.js (hunk 1 above)
Comment (RitikJaiswal75) [reply js/user.js:22]: Ok will do that

**Comment by RitikJaiswal75** [comment]
Made changes ap per the comments above

File: package.json [hunk 2]
```diff
@@ -15,5 +15,8 @@
   },
   "pre-commit": [
     "check"
-  ]
+  ],
+  "dependencies": {
+    "local-ssl-proxy": "^1.3.0"
+  }
```
Comment (prakashchoudhary07) [inline package.json:21]: Should this be in dependencies?

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: js/analytics.js [hunk 3]
```diff
@@ -1 +1,43 @@
...
+      for (
+        void 0 !== a ? (u = e[a] = []) : (a = 'posthog'),
+          u.people = u.people || [],
+          u.toString = function (t) {
+            var e = 'posthog';
+            return 'posthog' !== a && (e += '.' + a), t || (e += ' (stub)'), e;
+          },
+          u.people.toString = function () {
+            return u.toString(1) + '.people (stub)';
+          },
```
Comment (prakashchoudhary07) [inline js/analytics.js:28]: Is this file changed? If not then please remove it.

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (RitikJaiswal75) [reply package.json:21]: No that is just a local dependency just for testing

File: js/analytics.js (hunk 3 above)
Comment (RitikJaiswal75) [reply js/analytics.js:28]: Ok

Comment (RitikJaiswal75) [reply js/analytics.js]: This is the same file but with improved formatting. It's automatically changed by `npm run check` without which I cant commit.

**Pull request closed without merging**